authors/settings.py

```diff
@@ -81,8 +81,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```

Comment:
check out this question https://stackoverflow.com/questions/2099219/which-postgres-value-should-i-use-in-djangos-database-engine

Reply:
I had not seen that earlier, thank you. Since we are using django>1.9 i.e django 2.1 we should use 'django.db.bakends.postgresql'

Reply:
noted....changes made